content/en/blog/releases/release_v0.6.0.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Announcing OpenFunction 0.6.0: FaaS observability, HTTP trigger, and more"
+linkTitle: "Release v0.6.0"
+date: 2022-03-25
+weight: 95
+
+---
+
+[OpenFunction](https://github.com/OpenFunction/OpenFunction) is a cloud-native open source FaaS platform aiming to let you focus on your business logic. The OpenFunction community has been hard at work over the past several months preparing for the OpenFunction 0.6.0 release. Today, we proudly announce OpenFunction 0.6.0 is officially available. Thanks to the community for helping drive the new features, enhancements, and bug fixes.
+
+OpenFunction 0.6.0 brings notable features including function plugin, distributed tracing for functions, control autoscaling behavior, HTTP trigger to async function, etc. Meanwhile, OpenFuncAsync runtime definition has also been refactored. The core API has been upgraded from v1alpha1 to v1beta1. 
+
+### Distributed tracing for serverless functions
+
+When trying to understand and diagnose the distributed systems and microservices, one of the most effective methods is the stack trace. Distributed tracing provides a holistic view of the way that messages flow and distributed transaction monitoring for Serverless functions. The OpenFunction team collaborates with the Apache SkyWalking community to add FaaS observability which allows you to visualize function dependencies and track function invocations on SkyWalking UI. 
+
+![openfunction-tracing-skywalking](/images/docs/en/blogs/openfunction-tracing-skywalking.jpg)
+
+Going forward, OpenFunction will add more capabilities for serverless functions in logging, metrics, and tracing. You will be able to use Apache SkyWalking and OpenFunction to set up a full-stack APM for your serverless workloads out-of-the-box. Moreover, OpenFunction will support OpenTelemetry that allows you to leverage Jaeger or Zipkin as an alternative.
+
+### Support Dapr pub/sub and bindings
+
+Dapr bindings allows you to trigger your applications or services with events coming in from external systems, or interface with external systems. OpenFunction 0.6.0 adds Dapr output bindings to its synchronous functions which enables HTTP triggers for asynchronous functions. For example, synchronous functions backed by Knative runtime can now interact with middleware via Dapr components acting like the Async functions. See this [guide](https://github.com/OpenFunction/samples/tree/main/functions/knative/with-output-binding) for the quickstart sample.
```

Comment:
synchronous functions backed by Knative runtime can now interact with middleware via Dapr components acting like the Async functions.
=>
synchronous functions backed by the Knative runtime can now interact with middleware such as Dapr output binding or pubsub, and then an async function could be triggered by the events sent by the sync function.

Reply:
sync functions and async functions are separated, it doesn't act like an async function.
Dapr support to sync functions create a connection or trigger to async functions

Reply:
Updated. I think there are three places where we need to align the writing style:

- pubsub => pub/sub
- async => asynchronous
- sync => synchronous

The latter is more commonly used on other websites (Dapr, KEDA, etc.)